lib/taskana-core/src/main/java/pro/taskana/task/api/TaskService.java

```diff
@@ -490,4 +490,28 @@ TaskComment getTaskComment(String taskCommentId)
    */
   BulkOperationResults<String, TaskanaException> setPlannedPropertyOfTasks(
       Instant planned, List<String> taskIds);
+
+  /**
+   * Cancels a task.
```

Comment:
Cancels a task.
Cancellation means a task is obsolete from a business perspective an does not need to be completed anymore.

Reply:
I added the description